x-pack/legacy/plugins/apm/index.ts

```diff
@@ -88,22 +88,31 @@ export const apm: LegacyPluginInitializer = kibana => {
         catalogue: ['apm'],
         privileges: {
           all: {
-            api: ['apm', 'apm_write'],
+            api: ['apm', 'apm_write', 'actions-read', 'alerting-read'],
             catalogue: ['apm'],
             savedObject: {
-              all: [],
+              all: ['action', 'action_task_params'],
               read: []
             },
-            ui: ['show', 'save']
+            ui: [
+              'show',
+              'save',
+              'alerting:show',
+              'actions:show',
+              'alerting:save',
```

Comment:
Is it possible to grant users granular access, like `alerting:show` bu not `actions:show`?

Reply:
Possible for us? Or for admins?